src/main/java/com/minecolonies/coremod/generation/defaults/DefaultEnchanterCraftingProvider.java

```diff
@@ -0,0 +1,403 @@
+                .item(enchantedBook(Enchantments.BANE_OF_ARTHROPODS, 4), 5)
+                .item(enchantedBook(Enchantments.BLAST_PROTECTION, 4), 5)
+                .item(enchantedBook(Enchantments.DEPTH_STRIDER, 4), 5)
+                .item(enchantedBook(Enchantments.FEATHER_FALLING, 4), 5)
+                .item(enchantedBook(Enchantments.FIRE_ASPECT, 4), 5)
+                .item(enchantedBook(Enchantments.FIRE_PROTECTION, 4), 5)
+                .item(enchantedBook(Enchantments.FLAME, 4), 5)
+                .item(enchantedBook(Enchantments.FROST_WALKER, 4), 5)
+                .item(enchantedBook(Enchantments.INFINITY, 1), 5)
+                .item(enchantedBook(Enchantments.KNOCKBACK, 4), 5)
+                .item(enchantedBook(Enchantments.LOOTING, 4), 5)
+                .item(enchantedBook(Enchantments.POWER, 4), 5)
+                .item(enchantedBook(Enchantments.PROJECTILE_PROTECTION, 4), 5)
+                .item(enchantedBook(Enchantments.PUNCH, 4), 5)
+                .item(enchantedBook(Enchantments.RESPIRATION, 4), 5)
+                .item(enchantedBook(Enchantments.SHARPNESS, 4), 5)
+                .item(enchantedBook(Enchantments.SMITE, 4), 5)
+                .item(enchantedBook(Enchantments.SWEEPING, 4), 5)
+                .item(enchantedBook(Enchantments.UNBREAKING, 4), 5)
+                // plus new level 5 enchants
+                .item(enchantedBook(Enchantments.AQUA_AFFINITY, 5), 1)
+                .item(enchantedBook(Enchantments.BANE_OF_ARTHROPODS, 5), 1)
+                .item(enchantedBook(Enchantments.BLAST_PROTECTION, 5), 1)
+                .item(enchantedBook(Enchantments.DEPTH_STRIDER, 5), 1)
+                .item(enchantedBook(Enchantments.FEATHER_FALLING, 5), 1)
+                .item(enchantedBook(Enchantments.FIRE_ASPECT, 5), 1)
+                .item(enchantedBook(Enchantments.FIRE_PROTECTION, 5), 1)
+                .item(enchantedBook(Enchantments.FLAME, 5), 1)
+                .item(enchantedBook(Enchantments.FROST_WALKER, 5), 1)
+                .item(enchantedBook(Enchantments.INFINITY, 1), 1)
+                .item(enchantedBook(Enchantments.KNOCKBACK, 5), 1)
+                .item(enchantedBook(Enchantments.LOOTING, 5), 1)
+                .item(enchantedBook(Enchantments.MENDING, 1), 1)
+                .item(enchantedBook(Enchantments.POWER, 5), 1)
+                .item(enchantedBook(Enchantments.PROJECTILE_PROTECTION, 5), 1)
+                .item(enchantedBook(Enchantments.PROTECTION, 5), 1)
+                .item(enchantedBook(Enchantments.PUNCH, 5), 1)
+                .item(enchantedBook(Enchantments.RESPIRATION, 5), 1)
+                .item(enchantedBook(Enchantments.SHARPNESS, 5), 1)
+                .item(enchantedBook(Enchantments.SMITE, 5), 1)
+                .item(enchantedBook(Enchantments.SWEEPING, 5), 1)
+                .item(enchantedBook(Enchantments.UNBREAKING, 5), 1)
+                .item(enchantedBook(ModEnchants.raiderDamage, 2), 1)

```

Comment:
Another issue with this at present is that it violates the max level specified by the vanilla enchants -- for example:
* Aqua Affinity and Flame should only have level 1
* Punch only goes to level 2
* Respiration and Depth Strider only go to level 3
* Protection (all types) only go to level 4
(there are others)

This is not the first mod to allow "stronger" enchants than are possible via normal enchanting and crafting, and most will either cap their effect at the level they're supposed to (despite appearing higher) or will have an enhanced effect, which may or may not be intended.  But it's worth mentioning to consider if we want to fix this, or if it is intended.

Reply:
That is intended